### PR TITLE
Clan tag colouring

### PR DESCRIPTION
**Pull request by HardlySoftly**
As requested in https://github.com/FAForever/client/issues/157

Controllable by moderators using the Administrator menu

### CODE REVIEW

**Comment by HardlySoftly** [comment]
@Sheeo any chance you could do the server handling side of this?

**Comment by Sheeo** [comment]
Yes, we can't make it for 0.11 though. I'll get this done for the next release.

**Comment by HardlySoftly** [comment]
Any movement here?

**Comment by aeoncleanse** [comment]
Well, this it the first time I've really read through some Python top to bottom. From my limited understanding, this all seems pretty solid. 

:+1:

File: src/chat/channel.py [hunk 1]
```diff
@@ -51,13 +51,13 @@ def __init__(self, lobby, name, private=False, *args, **kwargs):
         BaseClass.__init__(self, lobby, *args, **kwargs)
 
         self.setupUi(self)
-        
+
         #Special HTML formatter used to layout the chat lines written by people
         self.lobby = lobby
         self.chatters = {}
-        
+
```
Comment (ChrisKitching) [inline src/chat/channel.py:58]: While my editor (and I) also hate ZeP's inability to do whitespace properly, polluting commits with these whitespace changes make it harder to spot the real changes. Please try not to do it (or, if you do want to, do it in a commit on its own so it doesn't give me hideous rebase conflicts when I try and review your work six months late :P )

**Comment by HardlySoftly** [comment]
How is this going?

**Comment by Sheeo** [comment]
No server support yet. It really hasn't been a priority for me -- go ahead and do the server implementation if you want.

File: src/chat/channel.py (hunk 1 above)
Comment (micheljung) [inline src/chat/channel.py:58]: While I agree, you can ignore whitespaces in any decent diff tool.
Even on github: https://github.com/FAForever/client/pull/265/files?w=1

**Pull request closed without merging**